Functions/PesterState.Tests.ps1

```diff
@@ -72,6 +72,36 @@ InModuleScope Pester {
         context "adding test result" {
             $p.EnterTestGroup('Describe', 'Describe')
 
+            it "times test accurately within 10 milliseconds" {
+                $p.EnterTest()
+                $Time = Measure-Command -Expression {
+                    Start-Sleep -Milliseconds 100
+                }
+                $p.LeaveTest()
+                $p.AddTestResult("result","Passed",$null)
+                $result = $p.TestResult[-1]
+                [int]($result.time.TotalMilliseconds) | Should -BeGreaterOrEqual ($Time.Milliseconds-10)
```

Comment:
Why are the casts to `[int]` needed?

Reply:
I forgot to remove them from an earlier test method.